DALK.PL_ANALYZER/Models/Matches/PlayedMatch.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace DALK.PL_ANALYZER.Models.Matches
+{
+    public class PlayedMatch : Match
+    {
+        public int HomePoints { get; set; }
+        public int AwayPoints { get; set; }
+        public int WinnerPoints
```

Comment:
This can be changed to expression bodies so it will take just one line, same for below props